controllers/drplacementcontrol_controller_test.go

```diff
@@ -818,7 +818,7 @@ func ensureDRPolicyIsNotDeleted(drpc *rmn.DRPlacementControl) {
 		err := apiReader.Get(context.TODO(), types.NamespacedName{Name: name}, drpolicy)
 		// TODO: Technically we need to Expect deletion TS is non-zero as well here!
 		return err == nil
-	}, timeout, interval).Should(BeTrue(), "DRPolicy deleted prematurely, with active DRPC references")
+	}).Should(BeTrue(), "DRPolicy deleted prematurely, with active DRPC references")
```

Comment:
This test looks very fragile - if there is some delay in the code, the drpolicy
may be deleted after the timeout even if we wait 10 seconds.

We need to verify instead that ramen handled the event and the drpolicy was not
deleted after the event was handled. Can we use the object generation for checking
if the event was handled?